### PR TITLE
Document that postMessage blocks the receiving thread

### PR DESCRIPTION
**Pull request by titoBouzout**
Well, I want to make a note about this because it says nowhere and measuring it was far from trivial.

### CODE REVIEW

File: files/en-us/web/api/worker/postmessage/index.html [hunk 1]
```diff
@@ -13,7 +13,7 @@
 ---
 <p>{{APIRef("Web Workers API")}}</p>
 
-<p>The <code><strong>postMessage()</strong></code> method of the {{domxref("Worker")}} interface sends a message to the worker's inner scope. This accepts a single parameter, which is the data to send to the worker. The data may be any value or JavaScript object handled by the <a href="/en-US/docs/Web/Guide/DOM/The_structured_clone_algorithm">structured clone</a> algorithm, which includes cyclical references.</p>
+<p>The <code><strong>postMessage()</strong></code> method of the {{domxref("Worker")}} interface sends a message to the worker's inner scope. This accepts a single parameter, which is the data to send to the worker. The data may be any value or JavaScript object handled by the <a href="/en-US/docs/Web/Guide/DOM/The_structured_clone_algorithm">structured clone</a> algorithm, which includes cyclical references.</p> Note this method blocks the communicating thread.
```
Comment (sideshowbarker) [inline files/en-us/web/api/worker/postmessage/index.html:16]:
```suggestion
<p>The <code><strong>postMessage()</strong></code> method of the {{domxref("Worker")}} interface sends a message to the worker's inner scope. This accepts a single parameter, which is the data to send to the worker. The data may be any value or JavaScript object handled by the <a href="/en-US/docs/Web/Guide/DOM/The_structured_clone_algorithm">structured clone</a> algorithm, which includes cyclical references.</p>
<p>Note this method blocks the communicating thread.</p>
```

**Review by sideshowbarker — CHANGES_REQUESTED** [review]

File: files/en-us/web/api/worker/postmessage/index.html [hunk 2]
```diff
@@ -14,6 +14,7 @@
 <p>{{APIRef("Web Workers API")}}</p>
 
 <p>The <code><strong>postMessage()</strong></code> method of the {{domxref("Worker")}} interface sends a message to the worker's inner scope. This accepts a single parameter, which is the data to send to the worker. The data may be any value or JavaScript object handled by the <a href="/en-US/docs/Web/Guide/DOM/The_structured_clone_algorithm">structured clone</a> algorithm, which includes cyclical references.</p>
+<p>Note this method blocks the communicating thread.</p>
```
Comment (sideshowbarker) [inline files/en-us/web/api/worker/postmessage/index.html:17]:
```suggestion
<p>Note this method blocks the thread which receives the message.</p>
```

**Review by sideshowbarker — CHANGES_REQUESTED** [review]

File: files/en-us/web/api/worker/postmessage/index.html [hunk 3]
```diff
@@ -14,6 +14,7 @@
 <p>{{APIRef("Web Workers API")}}</p>
 
 <p>The <code><strong>postMessage()</strong></code> method of the {{domxref("Worker")}} interface sends a message to the worker's inner scope. This accepts a single parameter, which is the data to send to the worker. The data may be any value or JavaScript object handled by the <a href="/en-US/docs/Web/Guide/DOM/The_structured_clone_algorithm">structured clone</a> algorithm, which includes cyclical references.</p>
+<p>Note this method blocks the thread which receives the message.</p>
```
Comment (sideshowbarker) [inline files/en-us/web/api/worker/postmessage/index.html:17]:
```suggestion

<p>Note this method blocks the thread which receives the message.</p>
```

**Review by sideshowbarker — CHANGES_REQUESTED** [review]

**Review by sideshowbarker — APPROVED** [review]

**Comment by sideshowbarker** [comment]
Thanks much, and congrats on landing your first docs change here — welcome aboard 🎉

**Comment by titoBouzout** [comment]
@sideshowbarker thank you for reviewing and the edit! <3

**Pull request merged**